install/linux/usr/share/odemis/hwtest/clspots-sem-diagnostic.odm.yaml

```diff
@@ -29,21 +29,22 @@ MultiBeamSEM: {
   }
 }
 
-#"Sample Stage": {
-#  class: pigcs.Bus,
-#  role: stage,
-#  init: {
-#    port: "/dev/ttyUSB0",
-#    # axis -> controller, channel, closed-loop?
-#    axes: {
-#      "z": [null, "Z", True]
-#    },
-#  },
-#}
-
 "Sample Stage": {
-    class: simulated.Stage,
+    class: pigcs.FakeBus,
     role: stage,
+    init: {
+        port: "/dev/ttyPIGCS",
+        axes: {
+            "x": [1, 1, True],
+            "y": [2, 1, True]
+        },
+        inverted: ["x", "y"],
+    },
+}
+
+"Optical Focus": {
```

Comment:
Since you renamed this, I think line 23 `children: {"z": "Sample Stage"},` should be `children: {"z": "Optical Focus"},`